drivers/vhost/dsp.c

```diff
@@ -0,0 +1,678 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
```

Comment:
I gave up on this file, you may want to split capabilities in e.g. open/close, configurations and streaming parts so that we don't have to scroll 700 lines of code. thanks!

FWIW, it's not just me, GitHub will not display the code by default since it's a 'large diff'. Tools don't lie.